Assets/Photon Unity Networking/Resources/PhotonServerSettings.asset

```diff
@@ -15,14 +15,14 @@ MonoBehaviour:
   AppID: 
   VoiceAppID: 
   ChatAppID: 
-  HostType: 2
+  HostType: 0
```

Comment:
Changes to the PhotonServerSettings.asset must be ignored, as otherwise multiplayer will break. There's a bug in PUN which will reset this file to the defaults, but for clarity the hostType must be 2, and JoinLobby must be 1

Reply:
roger, didn't notice as pretty much changed too many things on its own just by opening the proj, will fix and submit again